src/pages/Tap.vue

```diff
@@ -7,7 +7,6 @@
                 <h1>Type a voucher code</h1>
 
                 <form id="textVoucher" v-on:submit.prevent>
-                    <label for="voucherBox" id="lblVoucherBox">Type a voucher code</label>
```

Comment:
It is bad form, yes - meant to put an invisible one back in but I guess I got distracted!